### PR TITLE
Add disclaimer, and fix links

### PR DESCRIPTION
**Pull request by jaisnan**
*Issue #, if available:*

*Description of changes:*
Adds disclaimer that this project is not endorsed by Rust foundation and fixes license links.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +55,7 @@ You can find out more about Kani from the [Kani book](https://model-checking.git
 
 ## Contact
 
-For questions, suggestions or feedback, feel free to open an issue on the Kani page with the tag `stdlib-contest`  or contact us directly at [kani-developers@amazon.com](mailto:kani-developers@amazon.com).
+For questions, suggestions or feedback, feel free to open an [issue on the Kani page](https://github.com/model-checking/kani/issues) with the tag `stdlib-contest`  or contact us directly at [kani-developers@amazon.com](mailto:kani-developers@amazon.com).
```
Comment (celinval) [inline README.md:58]: Shouldn't they just create an issue in this repository instead?

**Review by celinval — COMMENTED** [review]
I just have one minor question

File: README.md (hunk 1 above)
Comment (jaisnan) [reply README.md:58]: I thought we wanted to save issues in this repository for challenges, and we can use a discussions tab to drive discussions about these challenges, and meta-level issues with the contest itself should be directed to Kani. lmk if you disagree.

File: README.md [hunk 2]
```diff
@@ -53,7 +55,7 @@ You can find out more about Kani from the [Kani book](https://model-checking.git
 
 ## Contact
 
-For questions, suggestions or feedback, feel free to open an issue on the Kani page with the tag `stdlib-contest`  or contact us directly at [kani-developers@amazon.com](mailto:kani-developers@amazon.com).
+For questions, suggestions or feedback, feel free to open an [issue on the Kani page](https://github.com/model-checking/verify-rust-std/issues) with the tag `stdlib-contest`  or contact us directly at [kani-developers@amazon.com](mailto:kani-developers@amazon.com).
```
Comment (celinval) [inline README.md:58]:
```suggestion
For questions, suggestions or feedback, feel free to open an [issue here](https://github.com/model-checking/verify-rust-std/issues).
```

File: README.md [hunk 3]
```diff
@@ -11,20 +11,22 @@ The goal is to have a contest to help verify the [Rust standard library](https:/
...
+
+NOTE: This work is not official, affiliated, or endorsed by the Rust project or Rust Foundation.
 * * *
 
 ## Contest Details
 
 Here are some details for the contest
 
-1. This repository will contain templates for [Issues](http://Insert link to Issues page), [pull requests](http://Link to PR page here) etc. that will be used to create new challenges for verifying the Rust standard library.
+1. This repository will contain templates for [Issues](https://github.com/model-checking/verify-rust-std/issues), [pull requests](https://github.com/model-checking/verify-rust-std/pulls) etc. that will be used to create new challenges for verifying the Rust standard library.
```
Comment (celinval) [inline README.md:22]:
```suggestion
1. This repository will contain templates for [Issues](https://github.com/model-checking/verify-rust-std/issues), and [pull requests](https://github.com/model-checking/verify-rust-std/pulls), that will be used to create new challenges for verifying the Rust standard library.
```

**Review by celinval — APPROVED** [review]

**Pull request merged**